<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSalir</class>
 <widget class="QDialog" name="dlgSalir">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Salir</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/salir.ico</normaloff>../img/salir.ico</iconset>
  </property>
  <widget class="QPushButton" name="btnSalirCancelar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalirSalir">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSalir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Â¿Deseas salir de CarTeis?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
